src/transition.js

```diff
@@ -76,13 +77,11 @@ export default class Transition {
   }
 
   _startTransitionTimer() {
-    var that = this;
-
     if (this.transitionIntervalId === null) {
-      that._startTransition();
-      this.transitionIntervalId = setInterval(function() {
-        that._startTransition();
+      this.transitionIntervalId = setInterval( () => {
```

Comment:
The space before the opening paren is failing an linting rule.

Reply:
@jungeunyoon good catch. cheers